### DISCUSSION TITLE
Annotation of training data in NER

### DISCUSSION THREAD

**Discussion by perin789** [explosion/spaCy#6721] [Help: Coding & Implementations]
Hi, I am annotating a document with custom entities, I realized that some entities have a lot of occurrences in the document hence having that many annotations, but some entities only have a few occurrences and annotations. The result of this is that the custom entities with few occurrences are not being identified in the testing stage by the Custom NER. 

Please, let me know if there is a way to solve this problem in spaCy? Would really appreciate the help. Thanks :)
@honnibal if you can help.

**Comment by ines** [comment]
The best way to solve this would be to just create more annotations of the entities that are less frequent in your data. In addition to that, if there are some instances of entities that are infrequent but that you know are pretty much _always_ an entity if they occur in the text, you can also use rules to augment the result, e.g. via the [`EntityRuler`](https://spacy.io/usage/rule-based-matching#entityruler).
